libphal/phal_dump.C

```diff
@@ -33,6 +33,11 @@ namespace dump
 using namespace openpower::phal::logging;
 using namespace openpower::phal::utils::pdbg;
 
+struct WriteSbeData {
+	uint64_t fapiRCode;
```

Comment:
How you are going to consume this data in odreport? like better to keep in key value format.

Reply:
Re-written the code. Writing the file in plain text so that opdreport can consume it directly, using the infra to write file was creating file in binary.